sklearn/inspection/_partial_dependence.py

```diff
@@ -279,6 +283,14 @@ def partial_dependence(
         The feature (e.g. `[0]`) or pair of interacting features
         (e.g. `[(0, 1)]`) for which the partial dependency should be computed.
 
+    sample_weight : array-like of shape (n_samples,), default=None
+        Sample weights for doing weighted means when averaging the model output.
+        If `None`, then samples are equally weighted. If `sample_weight` is not
+        `None`, then `method` will be set to `'brute'`.
+        Note that `sample_weight` does not change the individual predictions.
```

Comment:
```suggestion
        Note that `sample_weight` does not change the individual predictions nor ICE.
```

Reply:
When I said "individual predictions", I already meant the case of ICE. Do you think it wasn't clear? How can we write in a way that makes this explicit?

Reply:
```suggestion
        Note that `sample_weight` is ignored for `kind='individual'`.
```